src/Catty/ViewController/Continue&New/SceneTableViewController.m

```diff
@@ -101,41 +101,8 @@ - (void)addObjectAction:(id)sender
 {
     [self.tableView setEditing:false animated:YES];
     
-    [[[[[[[AlertControllerBuilder textFieldAlertWithTitle:kLocalizedAddObject message:[NSString stringWithFormat:@"%@:", kLocalizedObjectName]]
-          placeholder:kLocalizedEnterYourObjectNameHere]
-         addCancelActionWithTitle:kLocalizedCancel handler:^{
-        [self cancelAddingObjectFromScriptEditor];
-    }]
-        addDefaultActionWithTitle:kLocalizedOK handler:^(NSString *name) {
-        [self addObjectActionWithName:name];
-    }]
-       valueValidator:^InputValidationResult *(NSString *name) {
-        InputValidationResult *result = [Util validationResultWithName:name
-                                                             minLength:kMinNumOfObjectNameCharacters
-                                                             maxlength:kMaxNumOfObjectNameCharacters];
-        if (!result.valid) {
-            return result;
-        }
-        // Alert for Objects with same name
-        if ([[self.scene allObjectNames] containsObject:name]) {
-            return [InputValidationResult invalidInputWithLocalizedMessage:kLocalizedObjectNameAlreadyExistsDescription];
-        }
-        return [InputValidationResult validInput];
-    }] build]
-     showWithController:self];
-}
-
--(void)cancelAddingObjectFromScriptEditor
-{
-    if (self.afterSafeBlock) {
-        self.afterSafeBlock(nil);
-    }
-}
-
-- (void)addObjectActionWithName:(NSString*)objectName
-{
     [self showLoadingView];
-    [self.scene addObjectWithName:[Util uniqueName:objectName existingNames:[self.scene allObjectNames]]];
+    [self.scene addObjectWithName:@"default"];
```

Comment:
We should probably initialize with localized "Look" and uniqueName Util, in case the app is killed when the alert is open.